core/src/test/java/com/sequenceiq/cloudbreak/service/rdsconfig/HueRdsConfigProviderTest.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Please remove the wildcard import since it is usually not used anywhere else in the CB repo (and not a good practice either). Also applies to the other `*RdsConfigProviderTest` classes added below.